src/uicomponents/StatRadarPlot.tsx

```diff
@@ -62,6 +66,19 @@ function StatRadarPlot({nature, evs, stats, bossMultiplier=100}: {nature: Nature
                 //@ts-ignore
                 onRelayouting={onLayoutCallbacks}
                 data={[
+                    {
+                        type: "scatterpolar", // Hack to get hexagonal axis
+                        mode: "lines",
+                        line: {
+                            width: 1.5,
+                            color: isDark ? "#cccccc" : "#888888",
+                        },
+                        name: "axis",
+                        r: [1, 1, 1, 1, 1, 1, 1],
+                        theta: ticktexts,
+                        fill: "toself",
+                        fillcolor: "rgba(0, 0, 0, .1)",
+                    },
```

Comment:
A very minor thing, but it looks like the corners of the hexagon get very slightly cut off (presumably by half of their line width). If we want sharp corners, we could change `r` here to be a little smaller than 1, or we could edit the limits of the plot to be a little greater than 1.

Reply:
Looks good!